src/AuthManager.ts

```diff
@@ -170,4 +170,30 @@ export class AuthManager {
         });
     }
 
+    public async logout(): Promise<void> {
+        return new Promise((resolve, reject) => {
+            try {
+                const bearerToken = localStorage.getItem('access_token');
+                fetch(`${this.authServer}auth/logout`, {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        'Authorization': `Bearer ${bearerToken}`,
+                    },
+                }).then((response) => {
+                    if (response.status !== 200) {
+                        throw new Error('Failed to attempt logout')
+                    }
+                    localStorage.removeItem('access_token');
+                    localStorage.removeItem('refresh_token');
+                    resolve();
```

Comment:
shouldn't we also remove 
```
localStorage.removeItem('codeVerifier');
localStorage.removeItem('codeChallenge');
```
in case the user logged in with google

Reply:
`codeVerifier` and `codeChallenge` are automatically deleted after a successful login. At this point, `localStorage` only have `access_token` and `refresh_token`